.github/workflows/backwards_compatibility_marqo_orchestrator.yml

```diff
@@ -0,0 +1,174 @@
+      - name: Set up Python 3.9
+        uses: actions/setup-python@v3
+        with:
+          python-version: '3.9'
+          cache: "pip"
+
+      # Step to install the semver package
+      - name: Install semver
+        run: |
+          pip install semver
+
+
+      # Step to determine the target version
+      - name: Determine to_version
+        if: $${{ github.event_name != 'workflow_dispatch' && !github.event.inputs.to_version }} # If this condition is met it signifies
+        # that the run was triggered due to mainline push and was not manual. In that case we need to calculate to_version below. This is skipped otherwise.
+        id: get-version
+        run: |
+          VERSION=$(python tests/backwards_compatibility_tests/scripts/determine_to_version.py ${{ github.sha }})
+          echo "::set-output name=to_version::$VERSION"
+
+      # Step to generate the list of versions to test
+      - name: Generate version list #this code block just generates the from_version list and stores it in a versions variable as a list
+        id: versions
+        run: |
+          # Run the Python script and capture its output
+          VERSION_LIST=$(python tests/backwards_compatibility_tests/scripts/generate_versions.py ${{ steps.get-version.outputs.to_version }} ${{ env.MAX_VERSIONS_TO_TEST }})
+          echo "::set-output name=list::$VERSION_LIST" # setting the output variable
+
+      # Step to display the versions to test
+      - name: display versions
+        run: | 
+          echo "Versions to test: ${{ steps.versions.outputs.list }} against to_version: ${{ steps.get-version.outputs.to_version }}"
+
+      # Step to trigger execution workflows for each version combination
+      - name: Trigger execution workflows #this code block is then actually responsible for running the backwards_compatibility_marqo_execution.yml file with from_version / to_version combinations
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          for from_version in ${{ steps.versions.outputs.list }}
+          do
+            workflow_data="{\"ref\":\"${{ github.ref }}\",\"inputs\":{\"from_version\":\"$from_version\",\"to_version\":\"${{ steps.get-version.outputs.to_version }}\",\"to_version_tag\":\"${{ github.sha }}\""
+            
+            if [ -n "${{ github.event.inputs.to_image }}" ]; then
+              workflow_data="$workflow_data,\"to_image\":\"${{ github.event.inputs.to_image }}\""
+            fi
+            workflow_data="$workflow_data}}"
+            echo "printing workflow_data + " $workflow_data
+            curl -L \
+              -X POST \
+              -H "Accept: application/vnd.github+json" \
+              -H "Authorization: Bearer ${{ secrets.GITHUB_TOKEN }}" \
+              -H "X-GitHub-Api-Version: 2022-11-28" \
+              https://api.github.com/repos/adityabharadwaj198/marqo/actions/workflows/backwards_compatibility_marqo_execution.yml/dispatches \
```

Comment:
I guess this is an attempt to test the workflow that's not merged to mainline. I think what we can do is to create empty workflows and merge them in first.

Reply:
Removed this logic entirely so this is not needed anymore, started using`uses: ./.github/workflows/backwards_compatibility_marqo_execution.yml`